src/device/spu/spu.cpp

```diff
@@ -131,15 +137,13 @@ void SPU::step(device::cdrom::CDROM* cdrom) {
         bufferReady = true;
     }
 
-    if (control.irqEnable && irqAddress._reg < 0x200) {
-        static int cnt = 0;
+    uint32_t cd_left_address = capture_buffer_index;
```

Comment:
Please change to camelCase - I know it's not c++ like, but that's the convention I'm enforcing in this project.

I'll add style guide to CONTRIBUTING.md someday.

Reply:
Alright, all changes are now done.